src/components/Details/Legislation/VoteBar.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { FC, useState } from "react";
+import styled from "@emotion/styled";
+import { useMediaQuery } from "react-responsive";
+import { Link } from "react-router-dom";
+
+import { screenWidths } from "../../../styles/mediaBreakpoints";
+import Vote from "../../../models/Vote";
+import { strings } from "../../../assets/LocalizedStrings";
+
+const ColorBar = styled.div<{ width: string; zIndex: number; height: number }>((props) => ({
+  position: "absolute",
+  top: 0,
+  left: 0,
+  height: props.height,
+  width: props.width,
+  borderTopLeftRadius: "50em",
+  borderBottomLeftRadius: "50em",
+  borderTopRightRadius: "50em",
+  borderBottomRightRadius: "50em",
+  display: "flex",
+  flex: 1,
+  justifyContent: "space-between",
+  alignItems: "center",
```

Comment:
display, flex, justifyContent, alignItems can be removed because ColorBar doesn't have any children.